Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:7-alpine
+LABEL description="Alpine image to build nativfier apps"
+
+### Install wine depedency
+RUN apk add --no-cache \
+    wine \
+    freetype \
```

Comment:
Not sure if imagemagick is included within the base image, might have to add it here because we try to convert images to png